shop/src/pages/admin/_NewShop.js

```diff
@@ -100,6 +116,23 @@ const AdminNewShop = ({ shouldShow, onClose = () => {} }) => {
             {Feedback('name')}
           </div>
         )}
+        <div className="form-group">
+          <label>Store contact email</label>
+          <input {...input('storeEmail')} />
+          {Feedback('storeEmail')}
+          <div className="desc">
+            We&apos;ll use this address if we need to contact you about your
+            store.
+          </div>
+        </div>
+        <div className="form-group">
+          <label>Customer email</label>
+          <input {...input('supportEmail')} />
+          {Feedback('supportEmail')}
+          <div className="desc">
+            Your customers will see this address if you email them.
```

Comment:
nit: wonder if we should tweak this copy to clarify. the system actually sends the email automatically when new order are received for example. it's not really the merchant emailing themselves. Maybe @gullible1 can think of a better copy?

Reply:
@franckc good point about us the current automated emails.

@shahthepro can I trouble you to help change it to "Your customers will see this address when receiving emails about their order." please? Thanks (:

Reply:
Done.